content/en/blog/_posts/2023-01-06-unhealthy-pod-eviction-policy-for-pdb.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Eviction policy for unhealthy pods guarded by PodDisruptionBudgets"
+date: 2023-01-06
+slug: "unhealthy-pod-eviction-policy-for-pdbs"
+---
+
+**Authors:** Filip Křepinský (Red Hat), Morten Torkildsen (Google), Ravi Gudimetla (Apple)
+
+
+Ensuring the disruptions to your applications do not affect its availability isn't a simple
+task. With the introduction of _unhealthy pod eviction policy_ for [PodDisruptionBudgets](/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) (PDBs),
+this is going to change.
+In this article, we will dive deeper into what modifications were introduced for PDBs to
+give application owners greater flexibility in managing disruptions.
+
+## What problems does this solve?
+
+Eviction of pods respects PodDisruptionBudgets (PDBs). This means that the eviction of a pod
```

Comment:
(nit / optional change)
```suggestion
Eviction of pods respects PodDisruptionBudgets (PDBs). This means that a
[voluntary disruption](https://kubernetes.io/docs/concepts/scheduling-eviction/#pod-disruption)
to a Pod
```

Reply:
I suppose we could link that. What do you think about still having the eviction mentioned there?

```
This means that a requested [voluntary disruption](https://kubernetes.io/docs/concepts/scheduling-eviction/#pod-disruption)
via an eviction to a Pod, should not disrupt a guarded application and `.status.currentHealthy` of a PDB should not fall
below `.status.desiredHealthy`
```